src/test-jdk17/java/com/fasterxml/jackson/databind/records/RecordWithJsonIgnoreTest.java

```diff
@@ -64,9 +65,10 @@ public void testSerializeJsonIgnoreAndJsonPropertyRecord() throws Exception {
     }
 
     @Test
+    @JacksonTestFailureExpected
```

Comment:
If this was just to be merged, we'd need to now move this test class to under `tofix`.

Reply:
I agree, better to be moved, as I'm not sure how long it can take to figure out proper change, because simply removing that line I pointed to in the description would fix it for this test, but will break other tests that line was made for, so for now I will go with moving it so this PR can be merged.